src/views/tickets/rsvp.php

```diff
@@ -143,4 +152,8 @@
 $content = ob_get_clean();
 if ( $is_there_any_product ) {
 	echo $content;
+}else{
```

Comment:
@masteradhoc, I wonder if the logic on lines 89-97 would go nicely down here instead? The `Tickets are no longer on sale` message seems like a nice catch-all that would also satisfy the nostock message you have below! Thoughts?